src/CompanyService.Models.Dto/Models/DepartmentUserInfo.cs

```diff
@@ -1,10 +1,10 @@
-﻿using System;
-
-namespace LT.DigitalOffice.CompanyService.Models.Dto.Models
+﻿namespace LT.DigitalOffice.CompanyService.Models.Dto.Models
 {
-    public record DepartmentUserInfo
+    public record DepartmentUserInfo : UserInfo
```

Comment:
do you really want to do this?)

Reply:
I am about inheritance

Reply:
it is working now, if need be then we will rewrite in future